examples/fluids/qfunctions/stg_shur14.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+/// @file
+/// Implementation of the Synthetic Turbulence Generation (STG) algorithm
+/// presented in Shur et al. 2014
+//
+/// SetupSTG_Rand reads in the input files and fills in STGShur14Context. Then
+/// STGShur14_CalcQF is run over quadrature points. Before the program exits,
+/// TearDownSTG is run to free the memory of the allocated arrays.
+
+#ifndef stg_shur14_h
+#define stg_shur14_h
+
+#include <math.h>
+#include <ceed.h>
+#include <stdlib.h>
+#include "../src/setupstg_shur14.h"
```

Comment:
This (the struct definition) can be inlined here and `src/setupstg_shur14.h` can be deleted.

Reply:
Ok. I was going more on a DRY principle. For my own understanding, is there a downside to defining the struct in a single file and then `#include`ing it where needed?

Reply:
Personally, I'll put the struct in a separate header if its going to be used in multiple QFunctions and keep in with the QFunction header if it is only used in that one QFunction. Less opening files to see the whole picture that way.

Reply:
This header is currently only included in one place. It may be problematic to find such headers during JIT compilation so I'd say this indirection is not necessary -- just put the struct here.

Reply:
That makes sense to me. Didn't know if it had any JIT/backend implications I wasn't aware of. In this case the struct will only be used in this QFunction, so I'll move it to `qfunctions/stg_shur14.h` and then `#include` it for `src/setupstg_shur14.c`.

> ... and `src/setupstg_shur14.h` can be deleted.

> This header is currently only included in one place.

`SetupSTGContext()` (and `TearDownSTG()` if that ends up getting used) is going to be called externally, which is why I initially made `src/setupstg_shur14.h`. I now see that the other setup function instead put their function signatures into `navierstokes.h`. Should I follow that pattern?